API.md

```diff
@@ -1764,12 +1764,23 @@ Requires the string value to be a valid hexadecimal string.
 const schema = Joi.string().hex();
 ```
 
-#### `string.base64()`
+#### `string.base64([padding])`
+
+Requires the string value to be a valid base64 string; does not check the decoded value.
 
-Requires the string value to be a valid base64 string.
+* padding - optional parameter defaulting to `true` which will require `=` padding if `true` or make padding optional (`false`).
```

Comment:
If it's better to stick with a single parameter rather than an object, sounds good.